dev-tools/mage/settings.go

```diff
@@ -359,14 +357,15 @@ func PackageManifest() (string, error) {
 		return "", fmt.Errorf("retrieving agent flavors: %w", err)
 	}
 
-	return GeneratePackageManifest(BeatName, packageVersion, Snapshot, hash, commitHashShort, registry)
+	return GeneratePackageManifest(BeatName, packageVersion, Snapshot, FIPSBuild, hash, commitHashShort, registry)
```

Comment:
It's not a given that the global flag FIPSBuild will always be valid for usage in packaging. There will probably be a flag at spec level declaring if it's a fips spec or not, similar to what is being done in https://github.com/elastic/beats/pull/43001/files#diff-9e8bcd1359f1e06bbdf2c89db711c6c400ed4142cf9764ff03e6e963013122e5R631